Documentation/security/policy/language.rst

```diff
@@ -719,11 +721,15 @@ which is defined as follows:
 
         // PortProtocol specifies an L4 port with an optional transport protocol
         type PortProtocol struct {
-                // Port is an L4 port number. For now the string will be strictly
-                // parsed as a single uint16. In the future, this field may support
-                // ranges in the form "1024-2048"
+                // Port can be an L4 port number, or a name in the form of "http"
+                // or "http-8080".
```

Comment:
Is this also new, supporting arbitrary named ports? Do we have tests for this too?

Reply:
>Is this also new, supporting arbitrary named ports? Do we have tests for this too?

No, this is not new. This is a fragment of the previous comment. The only change in this comment is removing the conjecture about supporting port ranges as a hyphenated string (e.g. "8080-8090").